test_requirements.txt

```diff
@@ -1,3 +1,5 @@
 pyspark
 packaging
 pytest
+PyYAML
```

Comment:
I don't think this is necessary. please install by running

    python setup.py develop

to install all requirements.

Reply:
I have run python setup.py develop, still got some issues